src/events/SDL_mouse.c

```diff
@@ -1028,9 +1028,9 @@ void SDL_SendMouseWheel(Uint64 timestamp, SDL_Window *window, SDL_MouseID mouseI
     }
 
     // Accumulate fractional wheel motion if integer mode is enabled
-    if (mouse->integer_mode >= 2) {
-        mouse->wheel_x_frac = SDL_modff(mouse->wheel_x_frac + x, &x);
-        mouse->wheel_y_frac = SDL_modff(mouse->wheel_y_frac + y, &y);
+    if (mouse->integer_mode_flags & 2) {
```

Comment:
You'll need to update the documentation for this hint, and maybe define some constants instead of hardcoding 1 and 2.

Reply:
For what it's worth, `wheel_x_frac` is more intelligible to me than `integer_mode_residue_wx`. I would actually have called it `wheel_x_remainder`